lua/autorun/server/bounties.lua

```diff
@@ -46,7 +44,7 @@ RegisterChatCommand({'bounty','setbounty'},function(ply,arg)
 	local t = string.Explode(" ",arg)
 	local p = tonumber(table.remove(t))
```

Comment:
if arg is empty, table.remove here would be an error. I think if arg is empty (#t==0) you should print the usage, and where you currently print the usage you should print "Player (playername from string implode) not found"